### PR TITLE
Fix import in SPMD resnet

### PR DESCRIPTION
**Pull request by jonb377**
The SPMD resnet training test needs to import the new runtime package.

Also, I removed an outdated flag to enable virtual device.

### CODE REVIEW

File: test/spmd/test_train_spmd_imagenet.py [hunk 1]
```diff
@@ -67,6 +64,7 @@
 from torch_xla.distributed.fsdp.wrap import (recursive_wrap,
                                              transformer_auto_wrap_policy)
 from torch_xla.distributed.fsdp.utils import checkpoint_module
+from torch_xla import runtime as xr
```
Comment (JackCaoG) [inline test/spmd/test_train_spmd_imagenet.py:67]: hmm, why do we need to import if there is no code using it below?

Comment (jonb377) [reply test/spmd/test_train_spmd_imagenet.py:67]: We use it for building the device mesh: `    num_devices = xr.global_device_count()`. When migrating `pjrt` to `runtime`, the references were updated but the import was missed

**Review by JackCaoG — APPROVED** [review]
Thanks!

**Comment by JackCaoG** [comment]
I guess we can skip the CI, I don;t think this test is part of the CI(maybe it should be part of the TPU CI? running for couple steps for example).

**Pull request merged**